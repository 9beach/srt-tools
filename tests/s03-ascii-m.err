* 1 0,0: SHORT/FIXED BUT SHORT (0.524 -> 0.800)
  00:00:30,000 --> 00:00:30,524

* 2 1,19: SHORT/FIXED (0.100 -> 2.000)
  00:00:30,900 --> 00:00:31,000
  Lolita, light of my life,

* 3 1,24: SHORT/FIXED (0.575 -> 2.000)
  00:00:35,000 --> 00:00:35,575
  fire of my loins. My sin, my soul.

* 5 0,0: SHORT/FIXED BUT SHORT (1.000 -> 1.900)
  00:00:43,000 --> 00:00:44,000

* 6 2,58: OVERLAPPED/FIXED (5.469 -> 4.900)
  00:00:45,000 --> 00:00:50,469
  the tip of the tongue taking a trip of
  three steps down the palate to tap,

* 7 1,24: SHORT/FIXED (0.635 -> 2.000)
  00:00:50,000 --> 00:00:50,635
  at three, on the teeth. Lo. Lee. Ta.

* 8 0,0: SHORT/FIXED (1.000 -> 2.000)
  00:01:03,000 --> 00:01:04,000

