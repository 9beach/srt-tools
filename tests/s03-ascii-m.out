1
00:00:30,000 --> 00:00:30,799

2
00:00:30,900 --> 00:00:32,900
Lolita, light of my life,

3
00:00:35,000 --> 00:00:37,000
fire of my loins. My sin, my soul.

4
00:00:40,000 --> 00:00:42,000
<font color=red><i>Lo-lee-ta:</i></font>

5
00:00:43,000 --> 00:00:44,900

6
00:00:45,000 --> 00:00:49,900
the tip of the tongue taking a trip of
three steps down the palate to tap,

7
00:00:50,000 --> 00:00:52,000
at three, on the teeth. Lo. Lee. Ta.

8
00:00:63,000 --> 00:01:05,000
